app/views/conversations/index.haml

```diff
@@ -17,7 +17,7 @@
         #conversation_inbox
           .stream.conversations
             - if @conversations.count > 0
-              = render :partial => 'conversations/conversation', :collection => @conversations, :locals => {:authors => @authors, :ordered_participants => @ordered_participants, :unread_counts => @unread_counts}
+              = render :partial => 'conversations/conversation', :collection => @conversations, :locals => {:authors => @authors, :ordered_participants => @ordered_participants, :unread_counts => @unread_counts, :selected_conversation_id => (@conversation ? @conversation.id : -1)}
```

Comment:
`@conversation.try(:id) || -1`, but I'm not sure how I feel about using -1 as a magic value here, why not check for nil in the target code?